package.json

```diff
@@ -3,9 +3,10 @@
   "private": false,
   "version": "2.1.1",
   "description": "Node module to communicate with the Vultr API",
-  "main": "src/index.js",
+  "main": "dist/index.js",
```

Comment:
Thanks @spencerkordecki :)

This might break the next publish, until the`/dist` dir is included in the tarball that's pushed to npm.

I'll try and take a look later if you don't get to it, just wanted to warn you before the next `npm publish`